Specifications/monitoring.md

```diff
@@ -0,0 +1,316 @@
+# Monitoring
+
+- Version: 1
+
+## Introduction
+
+Pillarbox integrates with a monitoring platform that provides real-time and historical dashboards for:
+
+- Quality of Service (QoS): QoS refers to the performance level of a service, especially in terms of its transmission quality and service availability.
+- Quality of Experience (QoE): QoE is a user-centric measure that evaluates the overall experience of the user with a service.
+
+This article describes the flexible event model that allows our players to send data to our monitoring platform.
+
+## General Event Format
+
+Events provide data related to specific points of interests in the lifetime of a playback session. Three kinds of event types are available:
+
+- `START`
+- `ERROR`
+- Status events (`STOP`, `HEARTBEAT`)
+
+Associated information is provided in JSON payloads all having the same fixed top-level structure containing the following keys:
+
+| Key | Description | Format | Examples |
+| - | - | - | - |
+| `data` | Data associated with the event | JSON Dictionary | `{ ... }` |
+| `event_name` | The name of the event | `START`, `STOP`, `ERROR`, `HEARTBEAT` | `STOP` |
+| `session_id` | A unique identifier for the session | [UUID](https://www.itu.int/en/ITU-T/asn1/Pages/UUID/uuids.aspx) | `37b18444-76b6-4159-8539-d48ea5ecbc86` |
+| `timestamp` | The timestamp at the time the event is sent | [Unix timestamp](https://unixtime.org) in milliseconds | `1717665997932` |
+| `version` | The version of the JSON format | Number | 1 |
+
+> [!IMPORTANT]
+> All keys listed above are mandatory.
+
+All events associated with the same session must be assigned the same `session_id`. The `data` format associated with each event type is described in in more detail below.
+
+## Start Event `data`
+
+An event with the name `START` must be sent to signal the start of a playback session. This event conveys important static context information and is therefore required, no matter playback successfully starts or not:
```

Comment:
Could we highlight the **MUST**, **COULD**, **SHOULD** ?

Reply:
Done.